core/tests/protractor_utils/AdminPage.js

```diff
@@ -154,6 +154,54 @@ var AdminPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.getFeaturesTab = async function() {
+    await this.get();
+    const featuresTab = element(by.css('.protractor-test-admin-features-tab'));
+    await waitFor.elementToBeClickable(
+      featuresTab, 'Features tab is not clickable');
+    await featuresTab.click();
+    const featureFlagElements = element.all(
+      by.css('.protractor-test-feature-flag'));
+    await waitFor.visibilityOf(
+      featureFlagElements.first(), 'Feature flags not showing up');
+  };
+
+  this.getDummyFeatureElement = async function() {
+    const featureFlagElements = element.all(
+      by.css('.protractor-test-feature-flag'));
+    return featureFlagElements
+      .filter(async elem => {
+        return (await elem.element(by.css('h2.feature-name')).getText()) ===
+          'dummy_feature';
+      })
+      .first();
```

Comment:
Let's avoid the use of filters when dealing with async code. See point 2 in [this](https://github.com/oppia/oppia/wiki/End-to-End-Tests#anti-patterns) section of the wiki page.

Reply:
Done.